### PR TITLE
PIM-8438: display a flash message with the real error when attribute saving fails

### PR DESCRIPTION
**Pull request by rouliane**
**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/attribute/form/save.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                    );
+                    errorMessage = response.responseJSON[0] !== undefined ?
+                        response.responseJSON[0].message :
+                        errorMessage;
+                    break;
+                case 500:
+                    /* global console */
+                    const message = response.responseJSON ? response.responseJSON : response;
+
+                    console.error('Errors:', message);
```
Comment (pchasle) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/attribute/form/save.js]: Is there a reason to output the error in the browser console (especially as it's the same message for the flash message)?

**Comment by Doodoune** [comment]
Don't forget to add a changelog ;)

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/attribute/form/save.js (hunk 1 above)
Comment (rouliane) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/attribute/form/save.js]: Probably not, it's a copy paste from the parent file...

Comment (rouliane) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/js/attribute/form/save.js]: I removed it

**Pull request merged**